### PR TITLE
Get restaurants list from google sheet

### PR DESCRIPTION
**Pull request by yalefresne**
Requirement:  Google API Spread Sheet Enable
To test it you can run the following command:
`node -r dotenv/config dist/google.js`

Can close https://github.com/MyLittleParis/Random-Lunch/issues/4 if we validate google sheet as source.

### CODE REVIEW

File: src/google.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+// Doc : https://developers.google.com/sheets/api/reference/rest
+
+// If modifying these scopes, delete file which is in GOOGLE_TOKEN_FILE_PATH.
+const SCOPES = ["https://www.googleapis.com/auth/spreadsheets.readonly"];
+// The file which is in GOOGLE_TOKEN_FILE_PATH, stores the user"s access and refresh tokens
+// and is created automatically when the authorization flow completes for the first time.
+const TOKEN_PATH = path.join(process.cwd(), process.env.GOOGLE_TOKEN_FILE_PATH as string);
+const CREDENTIALS_PATH = path.join(process.cwd(), process.env.GOOGLE_CREDENTIALS_PATH as string);
+
+async function loadSavedCredentialsIfExist(): Promise<any|null> {
```
Comment (yalefresne) [inline src/google.ts:20]: I put `any` type here cause i was facing of type issue between `JSONClient` and `OAuth2Client` line 42 when i tried:
```
async function authorize(): Promise<OAuth2Client | JSONClient> {
```

Comment (yalefresne) [reply src/google.ts:20]: A solution will be:
```
async function loadSavedCredentialsIfExist(): Promise<JSONClient|OAuth2Client|null> {...}
async function authorize(): Promise<OAuth2Client|JSONClient> {...}
async function listRestaurants(auth: OAuth2Client|JSONClient): Promise<void> {...}
```
But the loadSavedCredentialsIfExist function never return OAuth2Client type.

Comment (Erwin-Felix-MLP) [reply src/google.ts:20]: do we need to store the credentials ? 
if not we could delete this method and we would not need to bother with this type definition warning

Comment (yalefresne) [reply src/google.ts:20]: Yes, because without the file, the authentication to the google api requires a manual action with a web browser.

**Review by Erwin-Felix-MLP — APPROVED** [review]

**Pull request merged**